GuruLoader/GuruLoader.cs

```diff
@@ -80,6 +80,25 @@ public class DisplayPortfolio {
     public IEnumerable<DisplayPosition> Positions { get; set; }
 }
 
+public class HyperPosition {
+    public string Name { get; set; }
+    public string ClassTitle { get; set; }
+    public string Cusip { get; set; }
+    public string PutCall { get; set; }
+    public double PercOfPortfolio { get; set; }
+    public int NumberGurusOwning { get; set; }
```

Comment:
This can probably be calculated from Gurus. That would also prevent mismatches where someone makes a mistake and says there's one guru owner but has two gurus in the Gurus list.